diff -urN zenity-2.5.90.orig/configure.in zenity-2.5.90/configure.in
--- zenity-2.5.90.orig/configure.in	2004-03-06 20:25:25.435235152 +0100
+++ zenity-2.5.90/configure.in	2004-03-06 20:25:43.080552656 +0100
@@ -36,7 +36,7 @@
 dnl Internationalization
 dnl ******************************* 
 
-ALL_LINGUAS="am ar az be bn ca cs cy da de el en_GB es et eu fa fi fr ga hi hr hu is it ja ko lt ml mn ms ne nl nn no pa pl pt pt_BR ru sk sl sq sr sr@Latn sv ta th uk vi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bn ca cs cy da de el en_GB es et eu fa fi fr ga hi hr hu is it ja ko lt ml mn ms nb ne nl nn pa pl pt pt_BR ru sk sl sq sr sr@Latn sv ta th uk vi zh_CN zh_TW"
 GETTEXT_PACKAGE=zenity-0.1
 AC_SUBST(GETTEXT_PACKAGE)
 AM_GLIB_GNU_GETTEXT
